### PR TITLE
1546 add deb permission note

### PR DESCRIPTION
**Pull request by asachs01**
replicating to 5.13, as I forgot this in the last PR.

### CODE REVIEW

**Review by cwjohnston — APPROVED** [review]

File: content/sensu-go/5.0/guides/systemd-logs.md [hunk 1]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.0/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.1/guides/systemd-logs.md [hunk 2]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.1/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.11/guides/systemd-logs.md [hunk 3]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.11/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.12/guides/systemd-logs.md [hunk 4]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.12/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.13/guides/systemd-logs.md [hunk 5]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.13/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.2/guides/systemd-logs.md [hunk 6]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.2/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.3/guides/systemd-logs.md [hunk 7]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.3/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.4/guides/systemd-logs.md [hunk 8]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.4/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.5/guides/systemd-logs.md [hunk 9]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.5/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.6/guides/systemd-logs.md [hunk 10]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.6/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.7/guides/systemd-logs.md [hunk 11]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.7/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.8/guides/systemd-logs.md [hunk 12]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.8/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

File: content/sensu-go/5.9/guides/systemd-logs.md [hunk 13]
```diff
@@ -70,3 +70,5 @@ You can use the following command to see what logrotate would do if it were exec
 {{< highlight shell>}}
 logrotate -d /etc/logrotate.d/sensu.conf
 {{< /highlight>}}
+
+_NOTE: For Ubuntu systems, there are some additional permissions required. After you've set up logging to disk on Ubuntu, ensure you run `chown -R syslog:adm /var/log/sensu` so that syslog will be able to write to that directory._
```
Comment (cwjohnston) [inline content/sensu-go/5.9/guides/systemd-logs.md]:
```suggestion
_NOTE: On Ubuntu systems, be sure to run `chown -R syslog:adm /var/log/sensu` so syslog can write to that directory._
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Suggesting a revised note for conciseness

**Review by Blue0ctober — APPROVED** [review]

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**